apps/web/playwright/lib/pageObject.ts

```diff
@@ -0,0 +1,19 @@
+import type { Page } from "@playwright/test";
```

Comment:
Plan to add more such interactions that could be shared across all types of Playwright tests for our app.